Sources/ArgumentParser/Parsable Properties/Option.swift

```diff
@@ -79,21 +99,67 @@ extension Option where Value: ExpressibleByArgument {
   ///   - parsingStrategy: The behavior to use when looking for this option's
   ///     value.
   ///   - help: Information about how to use this option.
+  @available(*, deprecated, message: "Use regular property initialization for default values (`var foo: String = \"bar\"`)")
   public init(
     name: NameSpecification = .long,
     default initial: Value? = nil,
```

Comment:
We can drop the default value for this parameter since you're adding `init(name:parsing:help:)`. I think overload resolution chooses that one over this because of the number of parameters, but it isn't strictly necessary anymore.

Reply:
Done in 28d7179